lib/Window.js

```diff
@@ -118,7 +118,7 @@ module.exports = (Item) => {
       const itemAtHotbarSlot = this.slots[hotbarSlot]
       if (item) {
         if (itemAtHotbarSlot) {
-          if (this.type === 'minecraft:inventory'/* || mcData.isNewerOrEqualTo('1.9')*/) {
+          if (this.type === 'minecraft:inventory'/* || mcData.isNewerOrEqualTo('1.9') */) {
```

Comment:
someone with more knowledge about PrismarineJS please implement this as it's needed to provide fully support for number clicks in 1.9 or above.